src/qmoperators/two_electron/FockBuilder.cpp

```diff
@@ -169,6 +169,7 @@ SCFEnergy FockBuilder::trace(OrbitalVector &Phi, const Nuclei &nucs) {
 
     // Reaction potential part
     if (this->Ro != nullptr) {
+        this->Ro->getHelper()->computeEnergies(Phi);
```

Comment:
With the above suggestion, here you would simply do:
```suggestion
        auto [Er_nuc, Er_el] = this->Ro->getHelper()->computeEnergies(Phi);
        auto Er_tot = Er_nuc + Er_el;
```
note I'm assuming the 0.5 is _included_ in the return value.

Reply:
it isn't now, but I can just add it :)

Reply:
I think this adds a bug in fockbuilder and SCFEnergy because of the c++ versioning (this wasnt available before c++17) I have added a small fix, but we should probably work on this in the future